app/components/game/create-product-dialog.tsx

```
@@ -1,4 +1,11 @@
 import { useState, type ReactNode } from "react";
+import {
+  useState,
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useState'. Consider to delete it or rename it.

'useState' is defined here:


(lint/suspicious/noRedeclare)